cfgov/cfgov/urls.py

```diff
@@ -74,243 +81,229 @@ def empty_200_response(request, *args, **kwargs):
         RedirectView.as_view(
-            url='/about-us/doing-business-with-us/%(path)s', permanent=True)),
+            url='/about-us/the-bureau/leadership-calendar/%(path)s',
+            permanent=True)),
+
+    re_path(r'^doing-business-with-us/(?P<path>.*)$', RedirectView.as_view(
+        url='/about-us/doing-business-with-us/%(path)s', permanent=True)),
 
-    url(r'^external-site/$', ExternalURLNoticeView.as_view(),
+    re_path(r'^external-site/$', ExternalURLNoticeView.as_view(),
         name='external-site'),
 
-    url(r'^subscriptions/new/$', govdelivery_subscribe, name='govdelivery'),
+    re_path(r'^subscriptions/new/$', govdelivery_subscribe,
+        name='govdelivery'),
 
-    url(r'^govdelivery-subscribe/', include([
-        url(r'^success/$',
-            TemplateView.as_view(
-                template_name='govdelivery-subscribe/success/index.html'),
+    re_path(r'^govdelivery-subscribe/', include([
+        re_path(r'^success/$', TemplateView.as_view(
+            template_name='govdelivery-subscribe/success/index.html'),
             name='success'),
-        url(r'^error/$',
-            TemplateView.as_view(
-                template_name='govdelivery-subscribe/error/index.html'),
+        re_path(r'^error/$', TemplateView.as_view(
+            template_name='govdelivery-subscribe/error/index.html'),
             name='user_error'),
-        url(r'^server-error/$',
-            TemplateView.as_view(
-                template_name='govdelivery-subscribe/server-error/index.html'),
+        re_path(r'^server-error/$', TemplateView.as_view(
+            template_name='govdelivery-subscribe/server-error/index.html'),
             name='server_error')],
         namespace='govdelivery')),
 
-    url(r'^regulation-comment/new/$', regsgov_comment, name='reg_comment'),
+    re_path(r'^regulation-comment/new/$', regsgov_comment, name='reg_comment'),
 
-    url(r'^regulation-comment/', include([
-        url(r'^success/$',
-            TemplateView.as_view(
-                template_name='regulation-comment/success/index.html'),
+    re_path(r'^regulation-comment/', include([
+        re_path(r'^success/$', TemplateView.as_view(
+            template_name='regulation-comment/success/index.html'),
             # 'core.views.comment_success',
             name='success'),
-        url(r'^error/$',
-            TemplateView.as_view(
-                template_name='regulation-comment/error/index.html'),
+        re_path(r'^error/$', TemplateView.as_view(
+            template_name='regulation-comment/error/index.html'),
             name='user_error'),
-        url(r'^server-error/$',
-            TemplateView.as_view(
-                template_name='regulation-comment/server-error/index.html'),
+        re_path(r'^server-error/$', TemplateView.as_view(
+            template_name='regulation-comment/server-error/index.html'),
             name='server_error')],
         namespace='reg_comment')),
```

Comment:
Same as above, this will need a [two-tuple with an app namespace](https://docs.djangoproject.com/en/2.2/ref/urls/#include):

```python
    re_path(r'^regulation-comment/', include(
        (
            [
                re_path(r'^success/$', TemplateView.as_view(
                    template_name='regulation-comment/success/index.html'),
                    # 'core.views.comment_success',
                    name='success'),
                re_path(r'^error/$', TemplateView.as_view(
                    template_name='regulation-comment/error/index.html'),
                    name='user_error'),
                re_path(r'^server-error/$', TemplateView.as_view(
                    template_name='regulation-comment/server-error/index.html'),
                    name='server_error')
            ],
            'reg_comment'
        ),
        namespace='reg_comment')
    ),
```

Reply:
Updated `regulation-comment` regular expression to a two-tuple with an application namespace